src/e2e/utils.py

```diff
@@ -0,0 +1,6 @@
+from src.net import Client
+
+
+class TestClient(Client):
```

Comment:
pytest thinks this is a test
```
src/e2e/utils.py:4
  /Users/jakubptak/work/starknet/starknet_python_sdk/src/e2e/utils.py:4: PytestCollectionWarning: cannot collect test class 'TestClient' because it has a __init__ constructor (from: src/e2e/deploy/deploy_test.py)
    class TestClient(Client):
```